network/src/commonMain/kotlin/com/wire/kalium/network/utils/NetworkUtils.kt

```diff
@@ -143,8 +145,13 @@ internal suspend inline fun <reified BodyType : Any> wrapKaliumResponse(performR
                         NetworkResponse.Error(KaliumException.Unauthorized(e.response.status.value))
                     }
 
-                    // TODO: try catch the parsing of error body
-                    else -> NetworkResponse.Error(kException = KaliumException.InvalidRequestError(e.response.body()))
+                    else -> try {
+                            e.response.body() as ErrorResponse
+                        } catch (_: NoTransformationFoundException) {
+                            ErrorResponse(e.response.status.value, e.response.status.description, "abc")
```

Comment:
should it be `"abc"`? 🤔

Reply:
Oh no, I was testing it and forgot to remove it 🙈 It's gone now